lib/authlogic/acts_as_authentic/login.rb

```diff
@@ -55,7 +54,11 @@ def find_by_smart_case_login_field(login)
 
         # @api private
         def find_with_case(field, value, sensitive)
-          Queries::FindWithCase.new(self, field, value, sensitive).execute
+          if sensitive
+            find_by(field => value)
+          else
+            find_by(arel_table[field].matches(value))
```

Comment:
I'm a little concerned about the performance of the `LIKE`/`ILIKE` operator (generated by `matches`). I worry that [LIKE might not use indexes well](https://use-the-index-luke.com/sql/where-clause/searching-for-ranges/like-performance-tuning), but I don't know enough about it.

Reply:
Huh... I don’t know about that either. I assume that the current `master` is also using `LIKE`/`ILIKE` under the hood, although (a) I could be wrong, and (b) that’s no reason to be dogmatic about continuing to do so, necessarily.